### PR TITLE
consequence type filter sorted alphabetically

### PR DESCRIPTION
**Pull request by kandaj**

### CODE REVIEW

File: src/js/eva-config.js [hunk 1]
```diff
@@ -131,23 +131,21 @@ consequenceTypes = [
...
-                    {acc: 'SO:0001621', name: 'NMD_transcript_variant', qtip: 'A variant in a transcript that is the target of NMD', leaf: true, checked: false, iconCls: 'no-icon', color: '#FF4500' },
-//                {acc: 'SO:0001582', name: 'initiator_codon_variant', qtip: 'A codon variant that changes at least one base of the first codon of a transcript',leaf: true,checked: false,  iconCls :'no-icon',color:'#FF4500' },
                     {acc: 'SO:0001626', name: 'incomplete_terminal_codon_variant', qtip: 'A sequence variant where at least one base of the final codon of an incompletely annotated transcript is changed', leaf: true, checked: false, iconCls: 'no-icon', color: '#FF00FF'},
+                    {acc: 'SO:0001821', name: 'inframe_insertion', qtip: 'An inframe non synonymous variant that inserts bases into in the coding sequence', leaf: true, checked: false, iconCls: 'no-icon', color: '#FF69B4'},
+                    {acc: 'SO:0001822', name: 'inframe_deletion', qtip: 'An inframe non synonymous variant that deletes bases from the coding sequence', leaf: true, checked: false, iconCls: 'no-icon', color: '#FF69B4'},
```
Comment (cyenyxe) [inline src/js/eva-config.js]: deletion should go before insertion

Comment (kandaj) [inline src/js/eva-config.js]: updated.

File: src/js/eva-config.js [hunk 2]
```diff
@@ -161,9 +159,10 @@ consequenceTypes = [
                 children: [
                     {acc: 'SO:0001623', name: '5_prime_UTR_variant', qtip: 'A UTR variant of the 5\' UTR', leaf: true, checked: false, iconCls: 'no-icon', color: '#7AC5CD'},
                     {acc: 'SO:0001624', name: '3_prime_UTR_variant', qtip: 'A UTR variant of the 3\' UTR', leaf: true, checked: false, iconCls: 'no-icon', color: '#7AC5CD'},
```
Comment (cyenyxe) [inline src/js/eva-config.js]: 3_prime_UTR_variant should go before 5_prime_UTR_variant.

File: src/js/variant-widget/filters/eva-consequence-type-filter-form-panel.js [hunk 3]
```diff
@@ -208,7 +208,7 @@ EvaConsequenceTypeFilterFormPanel.prototype = {
             model: 'Tree Model',
             proxy: {
                 type: 'memory',
-                data: this.consequenceTypes,
+                data:_.sortBy(this.consequenceTypes, 'name'),
```
Comment (cyenyxe) [inline src/js/variant-widget/filters/eva-consequence-type-filter-form-panel.js]: This doesn't seem to be working, given that the entries don't appear sorted? Or does it work only for the first level of the tree?

Comment (kandaj) [inline src/js/variant-widget/filters/eva-consequence-type-filter-form-panel.js]: It only works for the first level of the tree.

**Pull request merged**